.editorconfig

```diff
@@ -2,7 +2,6 @@ root = true
 
 [*]
 charset = utf-8
-end_of_line = crlf
```

Comment:
I think this was the line-ending issue culprit, combined with your git on Linux not being set to convert `\r\n` to `\n` on commit.

So I removed it and added a `.gitattributes`. I think this should resolve the issue going forward.

Reply:
It was a fresh install of git on a new linux machine so i had not yet configured the git environment. This is a good solution moving forward